package.json

```diff
@@ -59,14 +59,15 @@
   "types": "./index.d.ts",
   "scripts": {
     "lint": "populist lib/*.js lib/winston/*.js lib/winston/**/*.js",
-    "pretest": "npm run lint",
-    "test": "nyc --reporter=text --reporter lcov npm run test:mocha",
-    "test:mocha": "mocha test/*.test.js test/**/*.test.js --exit",
+    "test": "mocha './test/**/*.test.js'",
```

Comment:
```suggestion
    "test": "mocha",
```

Reply:
RE mocha npm scripts: 
I couldn't get these to work. If I don't define the `*.test.js` spec then it runs all files it finds in the `test/` dir, including plain `js` helper files. As soon as I define that recursive doesn't work as expected. As such these mocha commands are actully relying on the "globbing" behavior of `./test**/*.test.js`. I'm open to suggestions here but I could get nothing other than the above to function for this use case. Given this I will be disabling the `recursive` configuration as it's been deemed unnecessary.